example/res/layout/payment_activity.xml

```diff
@@ -1,31 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-              android:layout_width="fill_parent"
-              android:layout_height="fill_parent">
+<LinearLayout
```

Comment:
This is actually a no-op, but more efficient and easier to understand than using a RelativeLayout where each item just happens to be laid out under the one before it. This doesn't change the appearance of the app at all.